test/integration/int_server.cc

```diff
@@ -496,14 +496,14 @@ ServerCallbackHelper::ServerCallbackHelper(
       request_callback_(request_callback),
       close_callback_(close_callback) {
   if (request_callback) {
-    request_callback_ = [this, &request_callback](
+    instrumented_request_callback_ = [this](
```

Comment:
Ah!  Super cool.   I wonder if an equivalent fix would have been to capture the request_callback lambda by value?

Reply:
Didn't think of that, brain == fried. definitely worth a try!

Reply:
Great catch! If you fllow @duderino advice, could you try capture using expression? It might cost less while absolutely never more expensive. Something like
```
request_callback_ = [this, request_callback = std::move(request_callback) ](
```